Model/Config/EnableQueueComment.php

```diff
@@ -1,21 +0,0 @@
-<?php
-
-namespace Algolia\AlgoliaSearch\Model\Config;
-
-use Magento\Config\Model\Config\CommentInterface;
-use Magento\Framework\UrlInterface;
-
-class EnableQueueComment implements CommentInterface
-{
-    public function __construct(
-        protected UrlInterface $urlInterface
-    ) { }
-
-    public function getCommentText($elementValue)
-    {
-        $url = $this->urlInterface->getUrl('algolia_algoliasearch/queue/index');
-
-        return 'If enabled, all indexing operations (add, remove & update operations) will be done asynchronously using the CRON mechanism.<br><br/> Go to <a href="' . $url . '">Indexing Queue</a>. <br/> <br/>To schedule the run you need to add this in your crontab:<br>
```

Comment:
I understand this comment is out of data but I'm thinking it still might be useful to provide a link to the indexing queue from this page no? I don't believe that is achievable in straight XML in Magento.